src/radio-button-group.jsx

```diff
@@ -14,15 +14,42 @@ const RadioButtonGroup = React.createClass({
      * The css class name of the root element.
      */
     className: React.PropTypes.string,
+    /**
+     * Sets the default radio button to be the one whose
+     * value matches defaultSelected (case-sensitive).
+     * This will override any individual radio button with
+     * the defaultChecked or checked property stated.
+     */
     defaultSelected: React.PropTypes.string,
+
+    /**
+     * Where the label will be placed for all radio buttons.
+     * Options include "left" and "right" (case-sensitive).
+     * This will override any labelPosition properties defined
+     * for an individual radio button.
+     */
     labelPosition: React.PropTypes.oneOf(['left', 'right']),
```

Comment:
That make me think, the radio-button-group.jsx is not documented :grin:. I will fix this.